infra/scripts/node.toml

```diff
@@ -0,0 +1,111 @@
+[tasks.node]
+workspace = false
+dependencies = [
+    "node-down",
+    "node-init",
+    "node-network-create",
+    "cometbft-init",
+    "fendermint-deps",
+    "node-config",
+    "fendermint-start",
+    "cometbft-start",
+    "cometbft-wait",
+    "ethapi-start"
+]
+
+[tasks.node-network-create]
+env = { "NETWORK_NAME"="${NETWORK_NAME}"}
+extend = "docker-network-create"
+
+
+[tasks.node-init]
+dependencies = [
+    "node-clear",
+    "node-mkdir",
+]
+
+[tasks.node-clear]
+script="""
+echo clearing all IPC data
+rm -rf ${BASE_DIR}
+"""
+
+[tasks.node-mkdir]
+script="""
+echo creating directories: $BASE_DIR $FM_DIR $CMT_DIR
+mkdir -p $BASE_DIR
+mkdir -p $FM_DIR
+
+mkdir -p $CMT_DIR
+"""
+
+[tasks.node-down]
+# `dependencies` doesn't seem to work with `cleanup_task`.
+run_task = { name = [
+    "cometbft-stop",
+    "cometbft-rm",
+    "fendermint-stop",
+    "fendermint-rm",
+    "ethapi-stop",
+    "ethapi-rm",
+    "docker-network-rm"
+]}
+
+# This task create all necessary data structures to run Fendermint:
+# the genesis file with necessary entities and cryptographic keys.
+[tasks.node-config]
+dependencies = [
+    "node-script-new-genesis",
+    "node-script-new-key",
+    "node-script-new-account",
+    "node-script-add-validator",
+    "node-script-new-gateway",
+    "node-script-export-keys"
+]
+
+[tasks.node-script-new-genesis]
+cwd = "./target/release"
+script = """
+./fendermint genesis --genesis-file ${GENESIS_FILE} new --chain-name ${CHAIN_NAME} --base-fee ${BASE_FEE} --timestamp ${TIMESTAMP}
+"""
+
+[tasks.node-script-new-key]
+cwd = "./target/release"
+script="""
+mkdir -p $KEYS_DIR
+
+./fendermint key gen --out-dir $KEYS_DIR --name $KEY_NAME;
+chmod 600 ${PRIV_KEY_PATH}
+"""
+
+[tasks.node-script-new-account]
+cwd = "./target/release"
+script="""
+./fendermint genesis --genesis-file ${GENESIS_FILE} add-account --public-key ${PUB_KEY_PATH} --balance ${BALANCE}
+"""
+
+
+[tasks.node-script-add-validator]
+cwd = "./target/release"
+script = """
+./fendermint genesis --genesis-file ${GENESIS_FILE} add-validator --public-key ${PUB_KEY_PATH} --power 1
+"""
+
+[tasks.node-script-new-gateway]
+cwd = "./target/release"
+script = """
+./fendermint genesis --genesis-file ${GENESIS_FILE} ipc gateway --subnet-id /r0 \
+    --top-down-check-period 10 \
+    --bottom-up-check-period 10 \
+    --msg-fee 10 \
+    --majority-percentage 66
```

Comment:
Minor but strictly speaking 66% is less than 2/3.

Reply:
fixed